project4part2/presentation/description.tex

```diff
@@ -6,25 +6,13 @@ \section{Project Description}
 \frametitle{Project Description}
 \begin{itemize}
 \item The goal of our project is to develop a framework for
-  cryptographers to easily test their \cs s in a
+  cryptographers to test their \cs s in a
   secure environment.
 \item Users will be able to implement their
-  \cs{} securely in \cry, and upon testing it, they will
-  receive a report that will explain results of a particular
-  test as well as ways to improve the system.
-\end{itemize}
-\end{frame}
-
-\begin{frame}
-\frametitle{Description cont.}
-\begin{itemize}
-\item The will give details on various aspects of the
-  tested system. These details will include time (or
-  estimated time) required to encrypt, decrypt, and
-  cryptanalyze.
-\item When analyzing a cryptanalysis method, \cry{} will
-  hope to give some useful suggestions as to why the system
-  was broken in the given time, and what could be used to
-  improve it in further iterations.
+  \cs{} in \cry. As the \cs{} is tested, cry will
+  output a report that will outline certain details.
+\item These details include: estimated time required to
+  encrypt, decrypt, and crack the cipher using cryptanalysis
+  or brute-force.
```

Comment:
`brute-force` is an approach to `cryptanalysis`; a more clever approach might be possible